### PR TITLE
Link VMC to IP

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/InstancePricingApi.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
             @Query( "cloudProvider" ) int cloudProvider, @Query( "deprecated" ) Boolean deprecated
     );
 
+    @Deprecated
     @GET( "instancePricing" )
     Call<List<InstancePricingDTO>> getCompatible(
```
Comment (AndreaVecchi) [inline cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/InstancePricingApi.java]: MAJOR Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java [hunk 2]
```diff
@@ -64,6 +77,7 @@ public Integer getRam()
         return ram;
     }
 
+    @Deprecated
     public void setRam( Integer ram )
```
Comment (AndreaVecchi) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java]: MAJOR Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java [hunk 3]
```diff
@@ -49,11 +51,22 @@
...
+        this.instancePricing = instancePricing;
+    }
+
     public BigDecimal getCpu()
     {
         return cpu;
     }
 
+    @Deprecated
     public void setCpu( BigDecimal cpu )
```
Comment (AndreaVecchi) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java]: MAJOR Add the missing @deprecated Javadoc tag. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 3 major
* ![INFO][INFO] 3 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by malteo** [comment]
deprecated by #382

**Pull request closed without merging**